content/methods/discover/design-studio.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Design studio
+permalink: /methods/discover/design-studio/
+layout: layouts/method-card
+tags: methods
+redirect_from:
+  - design-studio/
+eleventyNavigation:
+  key: methods
+  parent: Discover
+  title: Design studio
+method:
+  title: Design studio
+  what: An illustration-based way to facilitate communication (and brainstorming) between a project team and stakeholders.
+  why: To create a shared understanding and appreciation of design problems confronting the project team.
+  timeRequired: 3–4 hours
+  category: discover
+  content: ./_includes/methods/markdown/design-studio.md
+  last_modified_at: example timestamp
```

Comment:
@jduss4 I think we can remove these `last_modified_at` keys. They don't appear to be used on the old site.